data/test/scenarios/wml_tests/ScenarioWML/EffectWML/test_remove_ability_by_filter.cfg

```diff
@@ -0,0 +1,124 @@
+# API(s) being tested: [effect]apply_to=remove_ability[filter_ability]
+##
+# Actions:
+# Two [chance_to_hit] abilities are added, setting the chance to 0 and 100 respectively.
+# The one setting the chance to 100 is removed, filtering by id and tag_name.
+# Alice attacks Bob.
+##
+# Expected end state:
+# Alice is unharmed, because all of Bob's attacks missed.
```

Comment:
There should be a `####` line at the top and bottom of this block. It's just decoration, but that's the template that Pentarctagon is using.

```
# All of Alice’s attacks hit.
# All of Bob’s attacks miss.
#####
```

Within the function itself, add some blank lines for readability, refer to `data/test/scenarios/macro_tests/test_force_chance_to_hit_macro.cfg` for suggestions about how much and where.

Reply:
done